Auth0Tests/ClaimValidatorsSpec.swift

```diff
@@ -66,7 +66,7 @@ class ClaimValidatorsSpec: IDTokenValidatorBaseSpec {
         describe("iss validation") {
             
             var issValidator: IDTokenIssValidator!
-            let expectedIss = "\(URL.a0_url(domain).absoluteString)/"
+            let expectedIss = "\(URL.httpsURL(from: domain).absoluteString)/"
```

Comment:
Not that it's breaking any of your tests, but this will end in `//`

Reply:
Good catch, I'll look into it.

Reply:
Indeed.

<img width="278" alt="Screen Shot 2021-11-30 at 13 47 34" src="https://user-images.githubusercontent.com/5055789/144091957-35511a5a-ecf2-42a5-976d-e8d477e2b8a8.png">

It does not break any tests because that constant is only used for checking presence and non-nil.

Reply:
I've removed the extra `/` in https://github.com/auth0/Auth0.swift/pull/557/commits/68be1ea56fb24dc4f01c7b8219dd5bccc941a568.